src/components/Missions.js

```diff
@@ -0,0 +1,39 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import Table from 'react-bootstrap/Table';
+import { getMission } from '../redux/missions/missions';
+import MissionOne from './missionOne';
+
+function Missions() {
+  const dispatch = useDispatch();
+  const missions = useSelector((state) => state.mission);
+
+  useEffect(() => {
+    dispatch(getMission());
+  }, [dispatch]);
```

Comment:
- [ ] Each time this component is rendered, the `getMission()` is dispatched. What this means is that there's an API call everytime this component renders, thereby fetching new data. As a result of this, if I join a mission, leave the missions page, and come back to it, my joined missions data is lost because the data is now fresh data.
In order to avoid this and to ensure that your missions data is fetched only once, please dispatch the `getMission()` only once in your `App` component 😎

Reply:
@codepantha thanks fixed